ush/cam/evs_href_prepare.sh

```diff
@@ -400,6 +402,7 @@ if [ "$data" = "mrms" ] ; then
       done
    else
       if [ "$SENDMAIL" = "YES" ] ; then
+	 echo "Warning:  No MRMS data available for ${VDATE}"
          export subject="MRMS Data Missing for EVS ${COMPONENT}"
          echo "Warning:  No MRMS data available for ${VDATE}" > mailmsg
          echo Missing file is $DCOMINmrms/MultiSensor_QPE_*.grib2.gz  >> mailmsg
```

Comment:
@BinbinZhou-NOAA I think we need a few changes here.

1) Lines 300. 342. 405 (and line 24 in the snowfall script) need to be written *outside* the SENDMAIL block.  The best place to do it is right after the else (i.e. between line 298 and 299 above).  Also, the warning statement needs to have the entire path of the file, so we have the missing file message and the full path to the file if SENDMAIL=NO (which it will be in operations).  

2) Also, Warning needs to be written as WARNING, in all capital letters.

Please make these changes in the href ush scripts.

@AliciaBentley-NOAA @ShelleyMelchior-NOAA for awareness.

Thanks!